python/dgl/data/chem/alchemy.py

```diff
@@ -189,10 +189,10 @@ def __init__(self, mode='dev', from_raw=False):
 
     def _load(self):
         if not self.from_raw:
-            with open(osp.join(self.file_dir, "%s_graphs.pkl" % self.mode), "rb") as f:
-                self.graphs = pickle.load(f)
-            with open(osp.join(self.file_dir, "%s_labels.pkl" % self.mode), "rb") as f:
-                self.labels = pickle.load(f)
+            self.graphs, label_dict = load_graphs(osp.join(self.file_dir, "%s_graphs.bin" % self.mode))
+            self.labels = label_dict['labels']
+            with open(osp.join(self.file_dir, "%s_smiles.pkl" % self.mode), 'rb') as f:
```

Comment:
If this is only a list of string(SMILES), I would suggest not using pickle since it's not readable for people and other application. A better choice is just write it to a txt file.

Reply:
Done.